environments/development/opg/sirius-daily/workflow.yml

```
@@ -0,0 +1,82 @@
+dag:
+  repository: moj-analytical-services/airflow-opg-etl
+  tag: v3.2.0-test-3
+  catchup: false
+  depends_on_past: false
+  is_paused_upon_creation: false
+  max_active_runs: 4
+  retries: 1
+  retry_delay: 150
+  schedule: "00 05 * * *"
+  start_date: "2025-04-30"
+  env_vars:
+    DATABASE: "sirius"
+    DATABASE_VERSION: "dev"
+    GITHUB_TAG: "v3.2.0-test-3"
+    ATHENA_DB_PREFIX: "opg"
+    START_DATE: "2025-04-30"
+    AWS_DEFAULT_REGION: "eu-west-1"
+    AWS_METADATA_SERVICE_TIMEOUT: "60"
+    AWS_METADATA_SERVICE_NUM_ATTEMPTS: "5"
+
+  secrets:
+    - sra_key
+    - survey_key
+    - survey_key_secret
+    - poa_satisfaction_details_id
+    - poa_satisfaction_responses_id
+    - deputyship_satisfaction_details_id
+    - deputyship_satisfaction_responses_id
```

```suggestion
secrets:
  - sra-key
  - survey-key
  - survey-key-secret
  - poa-satisfaction-details-id
  - poa-satisfaction-responses-id
  - deputyship-satisfaction-details-id
  - deputyship-satisfaction-responses-id
```
`secrets` is a top level key, and I've replaced `_` with `-` to conform to our naming scheme